src/screens/Settings/components/GeneralSettings/index.tsx

```diff
@@ -470,6 +474,25 @@ export default function GeneralSettings({
           </label>
         </span>
       )}
+      <span className="setting">
+        <label className={classNames('toggleWrapper', { isRTL: isRTL })}>
+          <ToggleSwitch
+            dataTestId="minimizeOnLaunch"
+            value={minimizeOnLaunch}
+            handleChange={toggleMinimizeOnLaunch}
+            title={t(
+              'setting.minimize-on-launch',
```

Comment:
Your variable is named `minimizeOnLaunch` and other translation titles follow camel case, but this is named `minimize-on-launch`. Shouldn't that also be `minimizeOnLaunch`?